.github/workflows/ci.yml

```diff
@@ -43,33 +42,30 @@ jobs:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v2
               with:
-                  node-version: 16.7.0
-            - uses: actions/cache@v2
-              with:
-                  path: "**/node_modules"
-                  key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
-            - run: yarn --frozen-lockfile
-            - run: yarn lint:sol
+                  node-version: 20.10.0
+                  cache: "npm"
+            - run: npm ci
+            - run: npm run lint:sol
 
     benchmarks:
         runs-on: ubuntu-latest
         strategy:
             fail-fast: false
             matrix:
-                solidity: ["0.7.6", "0.8.2"]
+                solidity: ["0.7.6", "0.8.24"]
                 include:
-                    - solidity: "0.8.2"
-                      settings: '{"viaIR":true,"optimizer":{"enabled":true,"runs":10000}}'
+                    - solidity: "0.8.24"
+                      settings: '{"viaIR":false,"optimizer":{"enabled":true,"runs":1000000}}'
```

Comment:
you mean disable right? because it doesn't work otherwise: https://github.com/safe-global/safe-smart-account/issues/544

this should be fixed in 1.5.0, and we should enable the optimizer on the main branch actually